src/rest_vol_group.c

```diff
@@ -542,6 +542,9 @@ RV_group_get(void *obj, H5VL_group_get_args_t *args, hid_t dxpl_id, void **req)
                         FUNC_GOTO_ERROR(H5E_SYM, H5E_SYSERRSTR, FAIL,
                                         "H5Gget_info_by_name request URL size exceeded maximum URL size");
 
+                    if (loc_info_out.GCPL_base64)
```

Comment:
In this function, `loc_info_out` should be initialized to 0 with `loc_info_out = {0};` and then this free call should be duplicated in the `done:` section so we don't miss cleaning up memory on failure. In that case, this free call here should set `loc_info_out.GCPL_base64` to NULL after freeing it so we don't accidentally try to double-free the pointer.